### PR TITLE
Bug 1711447 - Build for all iOS targets and bundle an xcframework

### PR DESCRIPTION
**Pull request by badboy**
Preparing for the xcframework build, first test run!

### CODE REVIEW

**Comment by badboy** [comment]
Just under 3 minutes extra build time isn't too bad IMO, and it only happens on release.

**Comment by badboy** [comment]
We can land this and I send a followup for the remaining work to push to [glean-swift](https://github.com/mozilla/glean-swift)

**Comment by badboy** [comment]
Decided to include the glean-swift release part already

File: .circleci/config.yml [hunk 1]
```diff
@@ -701,7 +717,7 @@ jobs:
           path: device_logs.zip
           destination: device_logs.zip
 
-  Carthage release:
+  iOS Framework release:
     macos:
       xcode: "12.4.0"
```
Comment (travis79) [inline .circleci/config.yml:722]: Can/should we update to 12.5?  Application Services just updated and Firefox-iOS is already on 12.5...

File: bin/publish-glean-swift.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+set -e
+
+# Strip any `v` prefix from `vX.Y.Z`
+NEW_VERSION=$(echo "$1" | sed 's/^v//')
+
+git clone https://github.com/mozilla/glean-swift glean-swift
+cd glean-swift
+bin/update.sh "$NEW_VERSION"
+git push -q https://${GLEAN_SWIFT_GITHUB_TOKEN}@github.com/mozilla/glean-swift.git main
+git push -q https://${GLEAN_SWIFT_GITHUB_TOKEN}@github.com/mozilla/glean-swift.git "$NEW_VERSION"
```
Comment (travis79) [inline bin/publish-glean-swift.sh:16]: Nice!  Do you think we will need to keep a separate repo for publishing forever?

**Review by travis79 — APPROVED** [review]
Just a couple of question, nothing to block landing on.

File: .circleci/config.yml (hunk 1 above)
Comment (badboy) [reply .circleci/config.yml:722]: Actually yeah, I thought I built the previous one on Xcode 12.4 for Focus, but actually it was 12.5 too

**Pull request merged**